buildenv/jenkins/variables/defaults.yml

```diff
@@ -83,16 +83,6 @@ build_discarder:
 restart_timeout:
   time: '5'
   units: 'HOURS'
-tests_targets:
-  sanity:
-    - sanity.functional
-  extended:
-    - extended.functional
-  default: &defaultTests
```

Comment:
Where are the tests targets set if they are removed here?

Reply:
We're not setting any default set other than "none" via L139 or L644.

Reply:
Are the test targets set in the job configuration, e.g PBT-All?

Reply:
Yes, all builds set them explicitly in the job config as a param. The exception being PR builds where we parse the comment and set them based on the user's request.